apis/keda/v1alpha1/scaledobject_webhook.go

```diff
@@ -92,23 +92,39 @@ func isRemovingFinalizer(so *ScaledObject, old runtime.Object) bool {
 
 func validateWorkload(so *ScaledObject, action string) (admission.Warnings, error) {
 	prommetrics.RecordScaledObjectValidatingTotal(so.Namespace, action)
-	err := verifyCPUMemoryScalers(so, action)
-	if err != nil {
-		return nil, err
+
+	verifyFunctions := []func(*ScaledObject, string) error{
+		verifyCPUMemoryScalers,
+		verifyTriggers,
+		verifyScaledObjects,
+		verifyHpas,
 	}
-	err = verifyScaledObjects(so, action)
+
+	err := verifyCPUMemoryScalers(so, action)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
Isn't this duplicated? I mean, it's here and also in verifyFunctions

Reply:
Good catch! It's a leftover from copy&paste

Reply:
Fixed